torch/csrc/autograd/python_variable.cpp

```diff
@@ -1507,9 +1507,10 @@ void concrete_dispatch_fn(const c10::impl::PyInterpreter*, const c10::OperatorHa
   py::handle torch_api_function = py::module::import("torch").attr("ops").attr(ns).attr(func_name);
   std::string module_name_str = "torch.ops." + ns_str;
 
-  // Pre-scan for arguments that match defaults
-  int64_t default_suffix_len = 0;
-  for (int64_t idx = arguments.size() - 1; idx >= 0; idx--) {
+  // Find the first valid idx that isn't defaulted.  E.g., if there
```

Comment:
Unless i'm misreading - isn't it the last index that isn't defaulted? (or -1 if every arg in the function is defaulted).
e.g.
```
def foo(a, b, c=1, d=1):
    pass
foo(1, 1, 1, 1)
index would be 1 in this case (`b`)